lib/sort-lines.js

```diff
@@ -52,6 +52,18 @@ function lineLengthReverseCompare(a, b) {
   return a.length > b.length ? -1 : 1;
 }
 
+function getVariableCharacters(line) {
+  return (line.match(/(.*)=/) || []).pop();
+}
+
+function variableLengthCompare(a, b) {
+  return getVariableCharacters(a).length > getVariableCharacters(b).length ? 1 : -1;
```

Comment:
[].pop().length will throw

Reply:
Yer I guess I could fallback to empty string in `getVariableCharacters`